marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs

```diff
@@ -259,9 +259,9 @@ function getEntityName(targetEntityType) {
 }
 
 function fallbackLegacyEntityLookup(targetEntityType) {
-  let targetArr = targetEntityType.split('/');
+  let targetArr = String(targetEntityType).split('/');
   let entityName = targetArr[targetArr.length - 1];
-  let tVersion = targetArr[targetArr.length - 2].split('-');
+  let tVersion = targetArr[targetArr.length - 2] ? targetArr[targetArr.length - 2].split('-') : '';
```

Comment:
What's an example of a targetEntityType that this is now handling?

Reply:
same as above. it was throwing a confusing error when targetEntityType was missing.